lua/include/packet.lua

```diff
@@ -5,17 +5,9 @@ require "headers"
 local dpdkc = require "dpdkc"
 local dpdk = require "dpdk"
 
-local eth = require "proto.ethernet"
-local arp = require "proto.arp"
-local ptp = require "proto.ptp"
-local ip = require "proto.ip"
-local ip6 = require "proto.ip6"
-local icmp = require "proto.icmp"
-local udp = require "proto.udp"
-local tcp = require "proto.tcp"
-
 local bor, band, bnot, rshift, lshift= bit.bor, bit.band, bit.bnot, bit.rshift, bit.lshift
 local istype = ffi.istype
+local write = io.write
```

Comment:
caching is usually just done for performance reasons and io.write is slow anyways